Add note to docs about updating licenses [FOEPD-184] (#271)
voxel51/fiftyone-teams-app-deploy: helm/fiftyone-teams-app/README.md

--from-file=license=./your-license-file
```

> [!TIP]
> When rotating the license, it may take a few minutes for the new secret
> to propagate to all pods. To ensure that the new license values take effect

> > immediately, you may need to restart the CAS and API pods.

Reviewer: ```suggestion
> immediately, you may need to restart the `teams-cas` and `teams-api` pods.
```

Reviewer 2: Maybe add a `kubectl rollout restart deploy -n your-namespace teams-cas teams-api`?

--from-file=license=./your-license-file
```

> > [!TIP]
> > When rotating the license, it may take a few minutes for the new secret
> > to propagate to all pods. To ensure that the new license values take effect
> > immediately, you may need to restart the `teams-cas` and `teams-api` services.
> > You can do this by deleting the pods, or by running the following command:
> > `kubectl rollout restart deploy -n your-namespace teams-cas teams-api`
> 

Reviewer 2: ```suggestion
 > [!TIP]
 > To ensure that the new license values take
 > effect immediately, you may need to restart the `teams-cas` and `teams-api`
 > services. </br>
 > You can do this by deleting the pods, or by running the following command:</br>
 > `kubectl rollout restart deploy -n your-namespace teams-cas teams-api`
```
